### PR TITLE
[FIXED issue/1545] added feedback to the user when a message content is copied to clipboard

### PR DESCRIPTION
**Pull request by Dekshut**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
Added success Alert after the message was copied to clipboard
**Is there anything you'd like reviewers to focus on?**
Auto-dissmis time and stylistic

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [x] Unit checks
- [X] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [X] My changes generate no new warnings (e.g. Sonar is happy)
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/Message.tsx [hunk 1]
```diff
@@ -32,12 +34,31 @@ const Message: React.FC<{ message: TopicMessage }> = ({
...
 }) => {
+  const AUTO_DISSMIS_TIME = 2000;
+  const dispatch = useAppDispatch();
   const [isOpen, setIsOpen] = React.useState(false);
   const { copyToClipboard, saveFile } = useDataSaver(
     'topic-message',
     content || ''
   );
 
+  const test = () => {
```
Comment (NelyDavtyan) [inline kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/Message.tsx:45]: it's not a good idea to name a variable 'test',change the name please

File: kafka-ui-react-app/src/lib/hooks/useDataSaver.ts [hunk 2]
```diff
@@ -1,13 +1,28 @@
 import { isObject } from 'lodash';
+import { alertAdded, alertDissmissed } from 'redux/reducers/alerts/alertsSlice';
+import { useAppDispatch } from 'lib/hooks/redux';
+
+const AUTO_DISSMISS_TIME = 2000;
```
Comment (Motherships) [inline kafka-ui-react-app/src/lib/hooks/useDataSaver.ts:5]: Typo
```suggestion
const AUTO_DISMISS_TIME = 2000;
```

File: kafka-ui-react-app/src/lib/hooks/useDataSaver.ts [hunk 3]
```diff
@@ -1,13 +1,28 @@
...
+      dispatch(
+        alertAdded({
+          id: subject,
+          type: 'success',
+          title: '',
+          message: 'Copied successfully!',
+          createdAt: Date.now(),
+        })
+      );
+      setTimeout(() => dispatch(alertDissmissed(subject)), AUTO_DISSMISS_TIME);
```
Comment (Motherships) [inline kafka-ui-react-app/src/lib/hooks/useDataSaver.ts:25]: Cool 
I will use same thing in my PR

**Review by Motherships — APPROVED** [review]

**Review by NelyDavtyan — APPROVED** [review]

File: kafka-ui-react-app/src/lib/hooks/useDataSaver.ts [hunk 4]
```diff
@@ -1,13 +1,28 @@
...
+import { alertAdded, alertDissmissed } from 'redux/reducers/alerts/alertsSlice';
+import { useAppDispatch } from 'lib/hooks/redux';
+
+const AUTO_DISMISS_TIME = 2000;
 
 const useDataSaver = (
   subject: string,
   data: Record<string, string> | string
 ) => {
   const copyToClipboard = () => {
```
Comment (abelgibayev) [inline kafka-ui-react-app/src/lib/hooks/useDataSaver.ts:11]: Move dispatch outside copyToClipboard, the hooks should be called only in other hook/component 1st level scope

**Review by abelgibayev — CHANGES_REQUESTED** [review]

**Review by abelgibayev — APPROVED** [review]

**Pull request merged**